Doc/library/os.rst

```diff
@@ -569,6 +569,32 @@ process and user.
       See the documentation for :func:`getgroups` for cases where it may not
       return the same group list set by calling setgroups().
 
+.. function:: setns(fd, nstype=0)
+
+   Reassociate thread with a namespace, see the :manpage:`setns(2)` man page for more details.
+
+   If *fd* refers to a ``/proc/[pid]/ns/`` link, ``setns()`` reassociates the
+   calling thread with the namespace associated with that link, subject to any
+   constraints imposed by the *nstype* argument (or any if ``0``).
+   Since Linux 5.8, *fd* may refer to a PID file descriptor obtained from
+   :func:`~os.pidfd_open`. In this case ``setns()`` reassociates the calling thread
+   into one or more of the same namespaces as the thread referred to by *fd*
```

Comment:
"one or more" -- does the caller choose which of the applicable namespaces the thread gets reassociated into? Or does it _always_ get reassociated into _all_ applicable namespaces, within the provided constraints?

Reply:
The next sentence is explains it: `subject to any constraints imposed by the *nstype*`

Reply:
&lt;nod&gt;, I caught that constraint from the following text, but there was some ambiguity to it on my first read. On rereading, I think it's all good.